src/main.js

```diff
@@ -90,13 +90,13 @@ async function main() {
     const actions = core.getInput('actions', { required: true });
 
     const actionsArr = actions.split(',');
-    actionsArr.forEach(item => {
-      testActions(item.trim());
-    });
+    for await (const action of actionsArr) {
+      testActions(action.trim());
+    }
```

Comment:
I wonder if `for await .. of` loop will take effect here? 🤔

Reply:
The `for await .. of` loop is used for iterating over async iterable objects. In our case, the "async iterable object" should be `actionsArr`.

However, the `actionsArr` is not a async iterable. It's just a common sync iterable array.

So I think we should simply use `await` in `for .. of` loop:

```ts
for (const action of actionsArr) {
  await testActions(action.trim());
}
```

Reply:
https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Statements/for-await...of

Reply:
```js
const arr = [1, 2, 3, 4];

const func = (foo) =>
  new Promise((resolve) => {
    setTimeout(() => {
      console.log(foo);
      resolve();
    }, 2000);
  });

const main = async () => {
  for await (const item of arr) {
    func(item);
  }

  for (const item of arr) {
    await func(item);
  }
};

main();
```